back-office/catalog/products/catalog-products-add-edit.md

```diff
@@ -81,11 +81,22 @@ If product with combinations was selected & combinations were generated, when I
 A drop-down next to the field is displayed when there are several languages installed \(disabled languages are diplayed in the drop-down\). It allows to choose in which language the field is displayed. Also, in the case of multilanguage, if an error occurs, then I must be warned in the message which language is problematic \([improvement 16299](https://github.com/PrestaShop/PrestaShop/issues/16299)\)
 * **Description**: The text can be formatted thanks to the TinyMCE editor \([improvement 14985](https://github.com/PrestaShop/PrestaShop/issues/14985)\)       
 A drop-down next to the field is displayed when there are several languages installed \(disabled languages are diplayed in the drop-down\). It allows to choose in which language the field is displayed. Also, in the case of multilanguage, if an error occurs, then I must be warned in the message which language is problematic \([improvement 16299](https://github.com/PrestaShop/PrestaShop/issues/16299)\)
-* **Quantity**: This field is not displayed for a product with combinations. This field is not displayed if stock management is disabled in Shop parameters &gt; Products settings.       
+* **Quantity**: 
+
+**1.7.7.0**:
+This field is not displayed for a product with combinations. This field is not displayed if stock management is disabled in Shop parameters &gt; Products settings.       
 If simple product is selected, under the quantity field, there is a “quantities” link. When you click on it, you switch to quantities tab \(for a standard product\) or to virtual product tab \(for a virtual product\).       
 When you update the field in basic settings tab, the same field in quantities \(for a standard product\) / virtual product tab is updated.       
 The quantity is displayed in front-office if the option “Display available quantities on the product page” in Shop parameters &gt; Product settings is set to yes.          
 This value is decremented when an order is placed except if the order status is Payement error. This value is incremented when the order status is cancelled, when you make a partial refund and check “Re-stock products” and when you return products and check “Re-stock products”.
+
+**1.7.8.0**:
+The quantity is now only a text in semibold that displays the quantity. The quantity can not be updated through the basic setting tab.
```

Comment:
Why not keeping the quantity field in basic settings tab ?

Reply:
Because we don't want the merchant to update the quantity on the product page. With the new wording on the quantity tab, we can bring awareness to the merchant.